usagov/vote-gov-drupal
# Vote-1041 -> Sprint 26 Review

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1041](https://bixal-projects.atlassian.net/browse/VOTE-1041)

## Description (optional)
After checking the work done in sprint 26 there were a few updates needed to be added to keep testing passing 

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down Branch 
2. run `lando rebuild -y`
3. in the root run `lando cy:testSuite` - please note there will be a fail in accessibility but work as been made for this
4. check that all tests outside of accessibility are passing
